src/components/password-modal/components/password-pin/password-pin.tsx

```diff
@@ -217,59 +195,17 @@ export class PasswordPinComponent extends React.Component<
         );
     }
 
-    public biometryAuth() {
```

Comment:
Keep in order to set `biometryType`.
You only need to remove `this.authenticate();`